source/_integrations/zha.markdown

```diff
@@ -402,6 +402,16 @@ Note that not all devices support binding as it depends on the Zigbee implementa
 
 Binding a remote directly to a bulb or group has the benefit of faster response time and smoother control. This greatly improves user feedback experience functions like dimming as the remote then directly dims the lightbulb and thus does not have to make the software roundtrip via the ZHA coordinator.
 
+## Zigbee backup and restore in ZHA
+
+Zigbee Home Automation (ZHA) integration now features Zigbee network backup, restore/recovery, and migrating between Zigbee coordinators since Home Assistant 2022.9 release. Backups are taken automatically however a single backup to file for easy download can also be manually created from the configuration page under Network Settings.
+
+After restoring a Home Assistant backup, you can re-configure ZHA and migrate to a new Zigbee Coordinator adapter without any loss of your settings or devices that were connected. This is helpful if your current radio fails or a new radio adapter type and model comes out that you may want to migrate to.
+
+ZHA saves the Zigbee network backups using the "[Open ZigBee Coordinator Backup Format](https://github.com/zigpy/open-coordinator-backup)" which is an open standard and hardware-independent non-proprietary specification for Zibgee coordinator backup using JSON formatting, meaning that backup files could possibly be cross-platform compatible if other Zigbee aplications use the same backup format.
```

Comment:
The backup is stored and exported in a zigpy-specific format. While it can be exported in in the open coordinator JSON format, this is not currently done and offers no benefits, since Zigbee2MQTT cannot actually import the backup.

Reply:
OK updated to reflect that.